### ISSUE TITLE
Should beta_classifier be multiplied with both loss_model and loss_z?

### ISSUE THREAD

**Issue by Natithan** [phlippe/CITRIS#1]
Hi,

Thanks for the cool work. A doubt I had: [in the CITRIS paper](https://arxiv.org/pdf/2202.03169.pdf#page=33), you explain just after Eq. 43 in C.3.3 that the full loss looks like: 
![image](https://user-images.githubusercontent.com/66829529/180225634-009ab730-1583-4717-b782-a013168456a7.png).

However, [in the code](https://github.com/phlippe/CITRIS/blob/f64fa27c19cd3bf20b4a4c0fc8de463197452c3b/models/citris_vae/lightning_module.py#L247), it looks like beta_classifier is only applied to loss_z: 
![image](https://user-images.githubusercontent.com/66829529/180225996-b89f3117-eb5e-4d72-b87a-4b7ad13329cf.png)

Should the code be `loss = loss + (loss_model + loss_z) * self.hparams.beta_classifier` instead of `loss = loss + loss_model + loss_z * self.hparams.beta_classifier`?

**Comment by phlippe** [comment]
Hi @Natithan, sorry for this confusion, it is indeed a bit misleading. However, in terms of experiments/results, it does not matter whether loss_model is multiplied by $\beta$ or not. The reason is that the parameters that are updated by `loss_model`, i.e. the target classifier's weights and biases, are solely influenced by `loss_model`. Since we use Adam as an optimizer, we (almost) do not change our parameter updates if we multiply the loss by a constant (the adaptive learning rate divides the gradients by the factor again up to the epsilon constant). Still, to be more consistent with the paper, I agree that it would be better to use `(loss_model + loss_z) * self.hparams.beta_classifier`. I will try to update it after ICML

**Comment by Natithan** [comment]
Another very small thing: [this line](https://github.com/phlippe/CITRIS/blob/f64fa27c19cd3bf20b4a4c0fc8de463197452c3b/models/shared/transition_prior.py#L378) has `target_params_soft = torch.softmax(target_params, dim=-1)`, which errors and I guess should be `target_params_soft = torch.softmax(self.target_params, dim=-1)` :)

**Comment by phlippe** [comment]
Thanks, that was indeed missing!
